simulation/utils.py

```diff
@@ -8,6 +8,7 @@ def wait_for_tx(tx_hash, web3, delay=5, wait_message=None):
     while True:
         try:
             reciept = web3.eth.getTransactionReceipt(tx_hash)
+            if reciept == None: raise Exception()
```

Comment:
Maybe add some message to the exception, like `Exception('Transaction reciept is empty')`?